server-base/src/main/java/org/apache/kylin/rest/msg/Message.java

```diff
@@ -65,6 +65,10 @@ public String getDELETE_NOT_READY_SEG() {
         return "Cannot delete segment '%s' as its status is not READY. Discard the on-going job for it.";
     }
 
+    public String getDELETE_READY_SEG_BY_UUID() {
+        return "Cannot delete segment by UUID '%s' as its status is READY or its Cube is READY. ";
```

Comment:
Remove the space at the end to keep the format uniform.

Reply:
fix.